### PR TITLE
DesktopStreamer: Grid-view like layout in apps listview

### PR DESCRIPTION
**Pull request by dnachbaur**

### CODE REVIEW

**Comment by dnachbaur** [comment]
FYI @eile

File: apps/DesktopStreamer/MainWindow.cpp [hunk 1]
```diff
@@ -72,6 +72,9 @@ const std::vector< std::pair< QString, QString > > defaultHosts = {
     { "DisplayWall 6th floor", "bbpav06.epfl.ch" }
 };
 
+const QString streamAll = "Stream all apps";
```
Comment (eile) [inline apps/DesktopStreamer/MainWindow.cpp]: Just "Stream all"?

**Comment by eile** [comment]
Looks nice, but my desktop image is too small: (cry)
screen shot 2016-03-31 at 8 47 09

**Comment by dnachbaur** [comment]
Hmm, dual-desktop setup I guess? I'll check what's happening there.

**Comment by eile** [comment]
Triple. Could be a fu since the main desktop display is not retina, but the builtin display is? For me is a low prio bug if it's working on the builtin display only.

**Comment by dnachbaur** [comment]
It only gets the image from the first desktop, but the size is all desktops. If streamed, the right part is black. So no multi-desktop support right now. It's part of https://github.com/BlueBrain/Deflect/issues/63 already.

**Comment by dnachbaur** [comment]
Looks like this now with 1 desktop
screen shot 2016-03-31 at 5 59 07 pm

**Comment by eile** [comment]
+1 from me, as implicitly said above.

**Comment by dnachbaur** [comment]
Next/last iteration
screen shot 2016-04-01 at 11 13 40 am

**Comment by eile** [comment]
Unselect/Select all is gone?

**Comment by rdumusc** [comment]
Sorry, merged already. @tribal-tec changed the behaviour a bit from the first version of the PR. Both versions work for me. I think this is also OK because there are different ways to select all windows (rectangle selection or cmd-a) and the 'stop streaming' stops all.

**Pull request merged**